konduit-serving-pipeline/src/main/java/ai/konduit/serving/pipeline/impl/data/helpers/ProtobufUtils.java

```diff
@@ -93,8 +106,130 @@ else if (value.type().equals(ValueType.NDARRAY)) {
 
                 item = DataProtoMessage.DataScheme.newBuilder().
                         setNdValue(pbNDArray).
+                        setTypeValue(ValueType.NDARRAY.ordinal()).
                         build();
             }
+            else if (value.type() == ValueType.DATA) {
+                JData jData = (JData)nextItem.getValue().get();
+                DataProtoMessage.DataMap dataMapEmbedded = serialize(jData.getDataMap());
+
+                item = DataProtoMessage.DataScheme.newBuilder().
+                        setMetaData(dataMapEmbedded).
+                        setTypeValue(ValueType.DATA.ordinal()).
+                        build();
+            }
+            else if (value.type() == ValueType.BOUNDING_BOX) {
+                BoundingBox boundingBox = (BoundingBox) nextItem.getValue().get();
+                DataProtoMessage.BoundingBox pbBox = null;
+                if (boundingBox instanceof BBoxCHW) {
+                    pbBox = DataProtoMessage.BoundingBox.newBuilder().
+                            setCx(boundingBox.cx()).setCy(boundingBox.cy()).
+                            setH(boundingBox.y2()-boundingBox.y1()).
+                            setW(boundingBox.x2()-boundingBox.x1()).
```

Comment:
I've merged PR today with height() and width() methods on BoundingBox interface

Reply:
Fixed.